runtime/runtime/Cargo.toml

```diff
@@ -34,6 +34,8 @@ default = []
 # Use this feature to enable counting of fees and costs applied.
 costs_counting = ["near-vm-logic/costs_counting", "near-vm-runner/costs_counting"]
 
+no_cache = []
```

Comment:
When `no_cache` is specified for `runtime` it should also make the dependency `near-vm-runner` use `no_cache`.